src/cleanfigure.m

```diff
@@ -96,7 +96,7 @@ function cleanfigure(varargin)
 
```

Comment:
You mean the cleanfigure around every test case? Indeed that's quite odd, but it has helped to find bugs in cleanfigure before.